src/docker.js

```diff
@@ -82,7 +82,8 @@ exports.getHostAddress = async () => {
     Entrypoint: 'sh',
     HostConfig: {
       AutoRemove: true,
-      NetworkMode: 'host'
+      NetworkMode: 'host',
+      UsernsMode: 'host'
```

Comment:
This function will fail on a system that is using user namespace remapping without this change. Since this is a host configuration setting I don't know how to create a test case that explicitly verifies this behavior. See https://docs.docker.com/engine/security/userns-remap/.